### PR TITLE
Added html for section 1

### LINKED ISSUE #1: HTML section 1
We need to create the layout of the first section

### PR DESCRIPTION
**Pull request by Viktoriya-Ulyanova**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -10,7 +10,26 @@
     <!-- шапка сайта  -->
     <header>Шапка сайта</header>
     <!-- секция 1 -->
-    <section>Секция 1</section>
+    <section class="section" id="warranty">
```
Comment (ghost) [inline index.html]: You need to replace the id value with "download"

File: index.html [hunk 2]
```diff
@@ -10,7 +10,26 @@
     <!-- шапка сайта  -->
     <header>Шапка сайта</header>
     <!-- секция 1 -->
-    <section>Секция 1</section>
+    <section class="section" id="warranty">
+      <div class="wrapper">
+        <div class="text-block">
+          <h1 class="title">Paeceful shopping mindful money</h1>
+          <p class="copy">See how we can help with making your shopping experience and money managment more ZEN</p>
```
Comment (ghost) [inline index.html]: If the element is large, it is better to move it to several lines
```
<p class="copy">
    See how we can help with making your shopping experience and money managment more ZEN
</p>
```

File: index.html [hunk 3]
```diff
@@ -10,7 +10,26 @@
...
+          <div class="links">
+            <a class="link" href="https://apple.com">
+              <img src="./assets/icons/link-apple.svg" alt="Apple icon">
+            </a>
+            <a class="link" href="https://google.com">
+              <img src="./assets/icons/link-google.svg" alt="Google icon">
+            </a>
+          </div>
+
```
Comment (ghost) [inline index.html]: Delete empty string

File: index.html [hunk 4]
```diff
@@ -10,7 +10,26 @@
...
     <!-- секция 1 -->
-    <section>Секция 1</section>
+    <section class="section" id="warranty">
+      <div class="wrapper">
+        <div class="text-block">
+          <h1 class="title">Paeceful shopping mindful money</h1>
+          <p class="copy">See how we can help with making your shopping experience and money managment more ZEN</p>
+          <div class="links">
+            <a class="link" href="https://apple.com">
+              <img src="./assets/icons/link-apple.svg" alt="Apple icon">
```
Comment (ghost) [inline index.html]: Recommended to close the single tag
```
<img src="./assets/icons/link-apple.svg" alt="Apple icon" />
```

**Review by ghost — COMMENTED** [review]
Fix

File: index.html (hunk 1 above)
Comment (Viktoriya-Ulyanova) [reply index.html]: look at the last(!) commit

File: index.html [hunk 5]
```diff
@@ -10,7 +10,27 @@
...
+          <h1 class="title">Paeceful shopping mindful money</h1>
+          <p class="copy">
+            See how we can help with making your shopping experience and money managment more ZEN
+          </p>
+          <div class="links">
+            <a class="link" href="https://apple.com">
+              <img src="./assets/icons/link-apple.svg" alt="Apple icon">
+            </a>
+            <a class="link" href="https://google.com">
+              <img src="./assets/icons/link-google.svg" alt="Google icon">
```
Comment (ghost) [inline index.html]: Close the single tag here too

File: index.html (hunk 4 above)
Comment (ghost) [reply index.html]: Close the single tag here too

**Review by ghost — COMMENTED** [review]
Fix

File: index.html (hunk 1 above)
Comment (Viktoriya-Ulyanova) [reply index.html]: fixed

**Review by ghost — APPROVED** [review]
Done

File: index.html [hunk 6]
```diff
@@ -10,7 +10,27 @@
     <!-- шапка сайта  -->
     <header>Шапка сайта</header>
     <!-- секция 1 -->
-    <section>Секция 1</section>
+    <section class="section" id="download">
```
Comment (nagorapro) [inline index.html]: Use BEM methodology
Use "download" class name

**Review by nagorapro — COMMENTED** [review]
Need fix

**Review by ghost — APPROVED** [review]
Done

**Pull request merged**